BuzzFeedNews/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+"author": {
+"name": "Bünyamin Öztürk",
+"id": "487515609815580672"
+},
+"service": "BuzzFeed News",
+"description": {
+"en": "BuzzFeed, Inc. is an American Internet media, news and entertainment company with a focus on digital media; it is based in New York City. BuzzFeed was founded in 2006 by Jonah Peretti and John S. Johnson III, to focus on tracking viral content.",
+"tr": "BuzzFeed, Inc. dijital medyaya odaklanan bir Amerikan İnternet medyası, haber ve eğlence şirketidir; New York'ta dayanmaktadır. BuzzFeed, 2006 yılında Jonah Peretti ve John S. Johnson III tarafından viral içeriklerin izlenmesine odaklanmak üzere kuruldu."
+},
+"url": [
+"www.buzzfeednews.com",
+"buzzfeednews.com"
+],
+"version": "1.3",
```

Comment:
Please change the version number every time you make an update.

Reply:
👌